components/Snackbar/src/private/MDCSnackbarOverlayView.m

```diff
@@ -27,14 +28,22 @@
 NSString *const MDCSnackbarOverlayIdentifier = @"MDCSnackbar";
 
 // The time it takes to show or hide the snackbar.
-NSTimeInterval const MDCSnackbarTransitionDuration = 0.5f;
+NSTimeInterval const MDCSnackbarEnterTransitionDuration = 0.15f;
+NSTimeInterval const MDCSnackbarExitTransitionDuration = 0.075f;
+NSTimeInterval const MDCSnackbarLegacyTransitionDuration = 0.5f;
```

Comment:
This code and the related animation logic is a good candidate for extracting into a Motion Interchange spec format in a follow-up change.